pkg/reconciler/openshift/tektontrigger/extension.go

```diff
@@ -32,11 +34,23 @@ func OpenShiftExtension(ctx context.Context) common.Extension {
 type openshiftExtension struct{}
 
 func (oe openshiftExtension) Transformers(comp v1alpha1.TektonComponent) []mf.Transformer {
+	// Updating the default values of runAsUser and runAsGroup to an empty string
+	// to ensure compatibility with OpenShift's requirements for managing these settings
+	// in Triggers Eventlistener containers SCC.
+	type triggersProperties struct {
```

Comment:
can we define the struct out of this function

Reply:
moved out of the function